examples/gno.land/r/demo/minidex/minidex.gno

```diff
@@ -0,0 +1,213 @@
+package minidex
```

Comment:
was this pkg supposed to be in `p/`?

Reply:
Maybe, but I prefer having more r/ than p/ at the moment. 

People are doing p/ that are not utilized. I definitely prefer the opposite.